1080i/Includes_MPAA.xml

```diff
@@ -82,8 +82,106 @@
         <value condition="String.IsEqual(ListItem.MPAA,Rated C16)">C16</value>
         <value condition="String.IsEqual(ListItem.MPAA,Rated C18)">C18</value>
         <value condition="String.IsEqual(ListItem.MPAA,Rated P)">P</value>
+		<value condition="String.IsEqual(ListItem.MPAA,Rated A)">A</value>
+		<value condition="String.IsEqual(ListItem.MPAA,Rated M/3)">M/3</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/6)">M/6</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/12)">M/12</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/14)">M/14</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/16)">M/16</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/18)">M/18</value>
         <value>$INFO[ListItem.MPAA]</value>
     </variable>
+	
+	<variable name="MPAA_Portugal">
+		<value condition="String.IsEqual(ListItem.MPAA,Rated A) | String.IsEqual(ListItem.MPAA,A) | String.IsEqual(ListItem.MPAA,Portugal A)">Portugal A.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/3) | String.IsEqual(ListItem.MPAA,M/3) | String.IsEqual(ListItem.MPAA,Portugal M3)">Portugal M3.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/6) | String.IsEqual(ListItem.MPAA,M/6) | String.IsEqual(ListItem.MPAA,Portugal M6)">Portugal M6.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/12) | String.IsEqual(ListItem.MPAA,M/12) | String.IsEqual(ListItem.MPAA,Portugal M12)">Portugal M12.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/14) | String.IsEqual(ListItem.MPAA,M/14) | String.IsEqual(ListItem.MPAA,Portugal M14)">Portugal M14.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/16) | String.IsEqual(ListItem.MPAA,M/16) | String.IsEqual(ListItem.MPAA,Portugal M16)">Portugal M16.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/18) | String.IsEqual(ListItem.MPAA,M/18) | String.IsEqual(ListItem.MPAA,Portugal M18)">Portugal M18.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated E) | String.IsEqual(ListItem.MPAA,E) | String.IsEqual(ListItem.MPAA,Australia E)">Australia E.png</value
```

Comment:
Missing `>` on `</value`

That's why it isn't working. 

I've got a script that generates the Includes_MPAA file properly based upon filenames, so no need to do this by hand.